### PR TITLE
[hotfix] Fix typos in comments.

### PR DESCRIPTION
**Pull request by yew1eb**
Hotfix some typos in comments.

### CODE REVIEW

File: flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java [hunk 1]
```diff
@@ -90,11 +90,11 @@
 	CompletableFuture<Acknowledge> releaseTaskManager(final ResourceID resourceId);
 
 	/**
-	 * Offers a slot to the {@link SlotPool}. The slot offer can be accepted or
+	 * Offers a slot to the {@link SlotPool}. The slot offers can be accepted or
 	 * rejected.
 	 *
-	 * @param taskManagerLocation from which the slot offer originates
-	 * @param taskManagerGateway to talk to the slot offerer
+	 * @param taskManagerLocation from which the slot offers originate
```
Comment (zentol) [inline flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java]: given that this method offers a single slot using singular makes more sense

File: flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java [hunk 2]
```diff
@@ -109,7 +109,7 @@
 	 * slot offers.
 	 *
 	 * @param taskManagerLocation from which the slot offers originate
-	 * @param taskManagerGateway to talk to the slot offerer
+	 * @param taskManagerGateway to talk to the slot offers
```
Comment (zentol) [inline flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java]: the previous version was correct

File: flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java [hunk 3]
```diff
@@ -90,11 +90,11 @@
...
 
 	/**
-	 * Offers a slot to the {@link SlotPool}. The slot offer can be accepted or
+	 * Offers a slot to the {@link SlotPool}. The slot offers can be accepted or
 	 * rejected.
 	 *
-	 * @param taskManagerLocation from which the slot offer originates
-	 * @param taskManagerGateway to talk to the slot offerer
+	 * @param taskManagerLocation from which the slot offers originate
+	 * @param taskManagerGateway to talk to the slot offers
```
Comment (zentol) [inline flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java]: the previous version was correct

**Comment by yew1eb** [comment]
@zentol  thanks for your attention to this PR, I had updated this PR according to your comments.

File: flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java [hunk 4]
```diff
@@ -108,7 +108,7 @@
 	 * individually accepted or rejected by returning the collection of accepted
 	 * slot offers.
 	 *
-	 * @param taskManagerLocation from which the slot offers originate
+	 * @param taskManagerLocation from which the slot offer originates
```
Comment (zentol) [inline flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java]: this method on the other hand offers multiple slots, so plural makes more sense.

**Comment by yew1eb** [comment]
:satisfied: ,  I addressed your comments.

**Pull request closed without merging**